bp/index.html

```diff
@@ -661,7 +662,16 @@ <h3>Data publication</h3>
 
         <p>Each of the four starting points outlined above have their own challenges, but working with plain text documents can be particularly tricky as you will need to parse the natural language to identify the <a>Spatial Things</a> and their properties before you can proceed any further. Natural Language Processing (NLP) is a complex topic in its own right and is beyond the scope of this best practice document. We will assume that you’ve already completed this step and have parsed any plain documents into structured data records of some kind.</p>
       </section>
-
+      <section id="scope-FAIR">
+	<h3>FAIR Principles</h3>
+	  <p>The FAIR Principles are described at <a xlink:href="https://www.go-fair.org/fair-principles/">FAIR Principles - GO FAIR</a>; they are widely adopted (or at least aimed for)
```

Comment:
`xlink:href` should be `href`.

Reply:
Whoops - a lot of this I tested offline (Notepad & browser), but I forgot this one!